CASMINST-4307-1.0 Initial checkin for site survey topic (#1662)
Cray-HPE/docs-csm: install/prepare_configuration_payload.md

| `--can-dynamic-pool` `10.103.11.128/25` | MetalLB dynamic pool on CAN |
| `--hmn-cidr` `10.254.0.0/17` | Override the default cabinet IPv4 subnet for River HMN |
| `--nmn-cidr` `10.252.0.0/17` | Override the default cabinet IPv4 subnet for River NMN |
| `--hmn-mtn-cidr` `10.104.0.0/17` | Override the default cabinet IPv4 subnet for Mountain HMN |
| `--nmn-mtn-cidr` `10.100.0.0/17` | Override the default cabinet IPv4 subnet for Mountain NMN |
| `--ntp-pools` `time.nist.gov` | External NTP pool(s) for this system to use |
| `--site-domain` `dev.cray.com` | Domain name for this system |
| `--site-ip` `172.30.53.79/20` | IP address and netmask for the PIT node `lan0` connection |
| `--site-gw` `172.30.48.1` | Gateway for the PIT node to use |
| `--site-nic` `p1p2` | NIC on the PIT node to become `lan0` |
| `--site-dns` `172.30.84.40` | Site DNS servers to be used by the PIT node |
| `--install-ncn-bond-members` `p1p1,p10p1` | NICs on each management node to become `bond0` |
| `--application-node-config-yaml` `application_node_config.yaml` | Name of `application_node_config.yaml` |
| `--cabinets-yaml` `cabinets.yaml` | Name of `cabinets.yaml` |

> | `--bgp-peers` aggregation | Override the default BGP peers, using aggregation switches instead of spines |

Reviewer: If the literal argument to use is "aggregation", it should be enclosed in backticks.

```suggestion
| `--bgp-peers` `aggregation` | Override the default BGP peers, using aggregation switches instead of spines |
```

* Several parameters (`can-gateway`, `can-cidr`, `can-static-pool`, `can-dynamic-pool`) describe the CAN (Customer Access network).
  * The `can-gateway` is the common gateway IP address used for both spine switches and commonly referred to as the Virtual IP address for the CAN.
  * The `can-cidr` is the IP subnet for the CAN assigned to this system.
  * The `can-static-pool` and `can-dynamic-pool` are the MetalLB address static and dynamic pools for the CAN.
  * The `can-external-dns` is the static IP address assigned to the DNS instance running in the cluster to which requests the cluster subdomain will be forwarded. The `can-external-dns` IP address must be within the `can-static-pool` range.
* Set `ntp-pools` to reachable NTP pools.
* The `application_node_config.yaml` file is required. It is used to describe the mapping between prefixes in `hmn_connections.csv` and HSM `subroles`. This file also defines aliases application nodes. For details, see [Create Application Node YAML](create_application_node_config_yaml.md).
* For systems that use non-sequential cabinet id numbers, use `cabinets-yaml` to include the `cabinets.yaml` file.

>   This file can include information about the starting ID for each cabinet type and number of cabinets which have separate command line options, but is a way to specify explicitly the id of every cabinet in the system.

Reviewer: ```suggestion
  This file can include information about the starting ID for each cabinet type and number of cabinets which have separate command line options, but is a way to specify explicitly the ID of every cabinet in the system.
```

See [Create Application Node YAML](create_application_node_config_yaml.md) for instructions about creating this file.

<a name="hmn_connections_json"></a>

### `hmn_connections.json`

The `hmn_connections.json` file is extracted from the HMN tab of the SHCD spreadsheet. The CSM release

> includes the `hms-shcd-parser` container which can be used on the PIT node booted from the LiveCD (RemoteISO

Reviewer: ```suggestion
includes the `hms-shcd-parser` container; this container can do the extraction on the PIT node booted from the LiveCD (RemoteISO
```

> or USB device) or a Linux system to do this extraction. Although some information in these files can be populated from site survey information, the SHCD prepared by HPE Cray Manufacturing is the best source of data for `hmn_connections.json`.

Reviewer: ```suggestion
or USB device) or on a Linux system. Although some information in these files can be populated from site
survey information, the SHCD prepared by HPE Cray Manufacturing is the best source of data for `hmn_connections.json`.
```
